models/src/it/java/io/sphere/sdk/reviews/ReviewIntegrationTest.java

```diff
@@ -0,0 +1,392 @@
+package io.sphere.sdk.reviews;
+
+
+import io.sphere.sdk.models.Versioned;
+import io.sphere.sdk.queries.PagedQueryResult;
+import io.sphere.sdk.queries.Predicate;
+import io.sphere.sdk.reviews.Review;
+import io.sphere.sdk.reviews.ReviewDraft;
+import io.sphere.sdk.reviews.commands.ReviewCreateCommand;
+import io.sphere.sdk.reviews.commands.ReviewUpdateCommand;
+import io.sphere.sdk.reviews.commands.updateCommands.SetAuthorName;
+import io.sphere.sdk.reviews.commands.updateCommands.SetScore;
+import io.sphere.sdk.reviews.commands.updateCommands.SetText;
+import io.sphere.sdk.reviews.commands.updateCommands.SetTitle;
+import io.sphere.sdk.reviews.queries.ReviewQuery;
+import io.sphere.sdk.test.IntegrationTest;
+import org.junit.Test;
+
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.Optional;
+
+import static io.sphere.sdk.customers.CustomerFixtures.withCustomer;
+import static io.sphere.sdk.products.ProductFixtures.withProduct;
+import static io.sphere.sdk.reviews.ReviewsFixtures.withReview;
+import static org.junit.Assert.*;
```

Comment:
I meant the english word exception (like "every can is open except one") not Javas Exception class.